### PR TITLE
COMPELSONLaboratories.MOBILeditPhoneManager version 10.6.1.28818

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98703)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

The installation isn't unattended - it's asking to install a phone driver:
image

Is there an installer switch to bypass this and have it install the phone driver automatically?

**Comment by upintheairsheep** [comment]
> Hi @upintheairsheep,
> 
> The installation isn't unattended - it's asking to install a phone driver: image
> 
> Is there an installer switch to bypass this and have it install the phone driver automatically?

Look at mobiledits website, is it possible to install the drivers first?

**Comment by stephengillie** [comment]
> Look at mobiledits website, is it possible to install the drivers first?

Were the driver package added to the WinGet manifests, and once dependency support is more stable, they could be added as a dependency to this package. That would allow them to be installed before this main package.

**Comment by stephengillie** [comment]
> 12/27/2023 8:45:54 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   2708FAB120644E3E405EB9B02B494B7B9B80DFF516AAF3A00667AEB50D72001C
Expected hash: 46d069f9a6e80320095c4ed22d5c49115b0118efe7e747fa38daaec6d3d3ede0

File: manifests/c/COMPELSONLaboratories/MOBILeditPhoneManager/10.6.1.28818/COMPELSONLaboratories.MOBILeditPhoneManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: COMPELSONLaboratories.MOBILeditPhoneManager
+PackageVersion: 10.6.1.28818
+Installers:
+- InstallerUrl: https://content.mobiledit.com/api/getproduct-mobiledit-win32
+  Architecture: x86
+  InstallerType: inno
+  InstallerSha256: 46D069F9A6E80320095C4ED22D5C49115B0118EFE7E747FA38DAAEC6D3D3EDE0
```
Comment (stephengillie) [inline manifests/c/COMPELSONLaboratories/MOBILeditPhoneManager/10.6.1.28818/COMPELSONLaboratories.MOBILeditPhoneManager.installer.yaml:10]:
```suggestion
  InstallerSha256: 2708FAB120644E3E405EB9B02B494B7B9B80DFF516AAF3A00667AEB50D72001C
```

**Pull request closed without merging**